### ISSUE TITLE
Helm autodiscovery produces multiple unexpected chart version bumps

### ISSUE THREAD

**Issue by gionn** [updatecli/updatecli#1475]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

A chart with multiple dependencies that needs a bump produce multiple chart version increments (one for each dependency that needs bumping), e.g.:

```
 name: alfresco-search-enterprise
 description: A Helm chart for deploying Alfresco Elasticsearch connector
 type: application
-version: 1.3.0
+version: 1.6.0
 appVersion: 3.3.0.1
 dependencies:
   - name: alfresco-common
-    version: 2.0.0
+    version: 2.1.0
     repository: https://alfresco.github.io/alfresco-helm-charts/
   - name: activemq
-    version: 3.1.0
+    version: 3.2.0
     repository: https://alfresco.github.io/alfresco-helm-charts/
     condition: activemq.enabled
   - name: elasticsearch
     repository: https://helm.elastic.co
-    version: 7.17.3
+    version: 8.5.1
     condition: elasticsearch.enabled
```

### Expected Behavior

Chart version should occur only one time in a single updatecli run

### Steps To Reproduce

1. Have a chart with multiple dependencies with newer versions available
2. Enable helm autodiscovery crawler via config
3. updatecli apply --experimental

### Environment

```markdown
- OS: macos 13.5
- updatecli: v0.57.0
```

**Comment by olblak** [comment]
Thanks @gionn  for reporting this issue, the problem is rooted in https://github.com/updatecli/updatecli/issues/693
I don't have yet a solution for this problem or at least a good one :( 

As a workaround, it's possible to disable the version bump by setting `versionincrement: none`

Since https://github.com/updatecli/updatecli/pull/1446 which was release in v0.56.0 the helm autodiscovery can also disable the the versionincrement

**Comment by gionn** [comment]
make sense, feel free to close this as a duplicate

**Comment by olblak** [comment]
@gionn based on your suggestion, I just opened https://github.com/updatecli/updatecli/pull/1526 
which disable versionincrement by default

**Comment by olblak** [comment]
The multiple version bump is now fixed with https://github.com/updatecli/updatecli/pull/1541 available in 0.59.0
Which seems to work as expected on my projects so I am now reverting https://github.com/updatecli/updatecli/pull/1572

To come back to the initial situation
